auth-web/src/components/auth/manage-business/EntityDetailsAlert.vue

```diff
@@ -0,0 +1,112 @@
+<!-- TODO:
+        get overlay colour
+        get alert icon colour
+        get header font and font size
+        compute array so that instead of mock arrays we get passed an array
+            of statuses then turn that array of statuses into an array of messages
+            then render those messages instead
+        align alert icon with bottom of status
+        change names of variables from status to something else
+         -->
+
+<template>
+   <v-tooltip top color="grey darken-4">
+     <template v-slot:activator="{ on }">
+       <v-icon :color=iconColour v-on="on">mdi-alert</v-icon>
+     </template>
+     <div class="py-2" :style="{'max-width': '300px'}">
+        <span>
+            {{ alertMessage }}
+        </span>
+        <ul>
+            <li v-for="message, i in detailMessages" :key="i">
+                {{ message.message }}
+            </li>
+        </ul>
+     </div>
+   </v-tooltip>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, nextTick, ref } from '@vue/composition-api'
+import { EntityDetailTypes } from '@/util/constants'
+import { PropType } from 'vue'
+
+export default defineComponent({
+  name: 'EntityDetailsAlert',
+  props: {
+    statuses: Array as PropType<Array<String>>
```

Comment:
this would be an array of EntityDetailTypes right?

Reply:
Yeah I'm assuming that we're going to pass an array of details (are they strings?) to the component. 
forgot to update the typing there

Reply:
Or are you just passing an array of enums (EntityDetailsAlert - enum) and this component determines the string based on that?

Reply:
right now I'm thinking that it's just oging to be an array of strings where each string is a status